File: src/AddIns/Misc/PackageManagement/Project/Src/EnvDTE/CodeElements.cs
// Copyright (c) AlphaSierraPapa for the SharpDevelop Team (for details please see \doc\copyright.txt)
// This code is distributed under the GNU LGPL (for details please see \doc\license.txt)

using System;
using System.Collections;

namespace ICSharpCode.PackageManagement.EnvDTE
{
	public interface CodeElements : IEnumerable
	{
		new IEnumerator GetEnumerator();
		
		int Count { get; }
		
		CodeElement Item(object index);
	}
}

File: src/AddIns/Misc/PackageManagement/SharpDevelop.EnvDTE/Src/CodeElements.vb
' Copyright (c) AlphaSierraPapa for the SharpDevelop Team (for details please see \doc\copyright.txt)
' This code is distributed under the GNU LGPL (for details please see \doc\license.txt)

Namespace EnvDTE
	Public Interface CodeElements
		Inherits IEnumerable
		
		ReadOnly Property Count() As Integer
		
		Function Item(index As Object) As CodeElement
		
		Function GetEnumerator() As IEnumerator
	End Interface
End Namespace